extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/config/GrpcServerNettyConfig.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.grpc.runtime.config;
+
+import java.time.Duration;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+@ConfigGroup
+public class GrpcServerNettyConfig {
+
+    /**
+     * Sets a custom keep-alive duration. This configures the time before sending a `keepalive` ping
+     * when there is no read activity.
+     */
+    @ConfigItem
```

Comment:
I'm not sure if it wouldn't be good for users to have some default set here. Possibly the default that is used if this is not set.

Reply:
I wonder if it would be even better to point to the place where the defaults are defined. In this case https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/internal/GrpcUtil.java#L215. I assume that there were some thoughts on setting this as defaults which need no additional reasoning in quarkus anymore. And with that on an update you would get the new defaults without maintaining it here again.

Reply:
I agree with @spike83. The default is managed upstream. I would prefer to keep it like this. 
Having our own default could introduce a divergence at some points. Until we see a reason to have another default value, let's use the upstream one.

Reply:
Note: Doing the same for other netty defaults.